ccd-definition/AuthorisationCaseField/cafcass.json

```diff
@@ -294,6 +294,13 @@
     "UserRole": "caseworker-publiclaw-cafcass",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "c2ApplicationType",
+    "UserRole": "caseworker-publiclaw-cafcass",
+    "CRUD": "CRU"
+  },
```

Comment:
could you just explain these extra permissions to me?

Reply:
this is new part of c2 upload journey, and permissions were missing for some users that are allowed to upload c2

Reply:
ok, can we just check for cafcass? I think I remember being told cafcass weren't going to be on the system when we were doing orders and directions

Reply:
There are existing cafcass permissions for `c2DocumentBundle` and `temporaryC2Document` which are parts of only C2 upload journey, so I assume this is also needed. Even if cafcass shouldn't have access to C2 upload, it's still not part of this PR.